userspace/libsinsp/container_engine/docker_linux.cpp

```diff
@@ -37,9 +39,14 @@ size_t docker_curl_write_callback(const char* ptr, size_t size, size_t nmemb, st
 	json->append(ptr, total);
 	return total;
 }
+#endif
 
+constexpr const cgroup_layout DOCKER_CGROUP_LAYOUT[] = {
+	{"/", ""}, // non-systemd docker
+	{"/docker-", ".scope"}, // systemd docker
```

Comment:
How confident are we about the `".scope"` suffix of systemd. 
this page here: 
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/resource_management_guide/sec-default_cgroup_hierarchies
seems to suggest we could have `".slice"` too. 

Will review the entire PR a little later. Wanted to capture this one thing first.